### ISSUE TITLE
CSS ID Linking to Specific nested Tab not working

### ISSUE THREAD

**Issue by flumpiey** [elementor/elementor#22784]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

The new experimental “Nested” Tabs allows the option to set a unique CSS ID which should be used for linking but, when trying to navigate to a specific tab using the ID of the tab item, it only shows the first tab and does not open the specific tab. The original non nested tab had this functionality.

### Steps to reproduce

1. Activate Nested Elements in Settings
2. Insert Tab element in page
3. Create CSS ID's for tabs
4. Create menu link to Tab ID's

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [X] I can reproduce this bug consistently following the steps above.

### System Info

- Wordpress - 6.2.2
- Elementor Pro - 3.13.2
- Elementor - 3.13.4

**Comment by nicholaszein** [comment]
Hello @flumpiey, 

Thank you for submitting your report! 🙏 

### ❌ While reviewing your report, we noticed it was missing some key pieces of information:

- 💾 **System info:** Please include the System Info report as outlined in our [guidelines](https://github.com/elementor/elementor/issues/new?assignees=&labels=status%2Fawaiting_triage&template=bug.yml). 
   - Get your System Info from your WordPress `Dashboard` > `Elementor` > `System info` [learn how to copy it here](https://elementor.com/help/hare-my-system-info/). We can't accept issues without it.

Without them, we simply can't help. 😕 

### 🧐 Please provide the missing information above so we can troubleshoot your issue faster and better.

Best regards

**Comment by massikage** [comment]
Hi,
I encounter exactly the same problem.
Is it fixed or not yet?
THANKS

**Comment by flumpiey** [comment]
> Hi, I encounter exactly the same problem. Is it fixed or not yet? THANKS

It still has not been fixed.

**Comment by massikage** [comment]
Is there any temporary solution for that ?
Thanks

Sent from Outlook for Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by digital-humanity** [comment]
We are experiencing this issue as well. 
It seems like there hasn't been an update, feedback or further input on this from Elementor's side?

**Comment by nicholaszein** [comment]
Hi @flumpiey!

Thank you for the updated System Info! 🙏 

Please allow us some time to investigate it, and get back to you with our findings. 🧐 

Best regards

**Comment by aaallan98** [comment]
Hi any workaround method to do this ?

**Comment by yeswycan** [comment]
> Hi @flumpiey!
> 
> Thank you for the updated System Info! 🙏
> 
> Please allow us some time to investigate it, and get back to you with our findings. 🧐
> 
> Best regards

We are 2 months later, any update about this problem ? Thanks !

**Comment by Caferohe** [comment]
@nicholaszein Unfortunately, the issue still persists. When I click on the tab title, it highlights the title box but doesn't open the tab content. I haven't come across any recent updates or changes that have resolved this problem. If there are any new developments or solutions, I would appreciate any information or assistance.

**Comment by Caferohe** [comment]
Hello @flumpiey 
I was wondering if you have found any alternative measures or solutions to address this issue. Your comments would be greatly appreciated.

Thank you in advance.

**Comment by sanket-haval-uniqueschools** [comment]
@nicholaszein Any update on this issue? 
I am also facing the same issue as jump to links doesn't navigate to the Tabs content

**Comment by adamr415** [comment]
Same issue, seems like this is an oversight since the CSS ID feature is there and it says it should work from the information provided in the [Tab Widget documentation](https://elementor.com/help/tabs-with-nested-containers/)

> CSS ID Tag` individual elements on a page. This allows you to link to this specific element.

**Comment by zanga7** [comment]
Experiencing the same. Looking forward to hearing a solution.

**Comment by JennyPower** [comment]
Same

**Comment by yeswycan** [comment]
Any update about this problem ? It's a shame this problem is still not solved since you talk about that in your doc, you sell something who is not working actually .... @nicholaszein @KingYes

**Comment by nicholaszein** [comment]
Hello everyone.

After consulting our devs, we concluded this is not a bug. This is a feature that isn't supported at the moment.

The reason why I didn't closed this issue was because it has a lot of comments.

Still,this isn't a bug, and requires further development to work as intended since this is not the native behavior.

There is no ETA for when such feature will be added.

**Comment by adamr415** [comment]
Thats fair @nicholaszein but then update the docs so that it doesn't give people the sense that the functionality would work like that. I spent hours trying to figure out what I did wrong.

> CSS ID Tag` individual elements on a page. This allows you to link to this specific element.

**Comment by nicholaszein** [comment]
@adamr415 there is nothing in the documentation that says you can link to the tab using CSS IDs already.

I'll soon close this issue as there is already a feature request that you can vote and discuss. Issues are for bug reports only.

**Comment by LabDevGit** [comment]
> @adamr415 there is nothing in the documentation that says you can link to the tab using CSS IDs already.
> 
> I'll soon close this issue as there is already a feature request that you can vote and discuss. Issues are for bug reports only.

But the document actually says this

"CSS ID: Tag individual elements on a page. This allows you to link to this specific element."
https://elementor.com/help/tabs-with-nested-containers/

**Comment by mstudioIL** [comment]
It still not working, I added custom CSS ID, and it still not shown, I reload many times, clear and regenerate CSS and data from the tools.

**Comment by djaysan** [comment]
guys if you still need to achieve this, i found a fix online that worked like a treat...
https://element.how/elementor-open-specific-tab-toggle-accordion/

**Comment by yeswycan** [comment]
> guys if you still need to achieve this, i found a fix online that worked like a treat... https://element.how/elementor-open-specific-tab-toggle-accordion/

yes i've write a little post about that few months ago and it work well :)
https://josselynjayant.fr/en/custom-css-id-link-to-elementor-nested-tabs/

**Comment by buckmanhands** [comment]
This basic functionality is absolutely necessary to link to content in a tab or accordion. We should not need to pay for any of the other add-on plugins which solved this issue or add custom javascript to make this happen. Please bake this into your very expensive Elementor Pro at minimum.

**Comment by buckmanhands** [comment]
Here is the code I am using to achieve this. I put it into my Document Ready function. This is for the accordions, I am sure if you looked for a tab it would also function properly. I only need this for external links into my website, more would need to be done to make it work when the page is already loaded. This is only meant to jumpstart you. If you have complicated pages with other uses for the url hash then it could cause code collisions, I know my site and it is not an issue for me since URL hash is not widely used.

```
// check if anchor link in url
var hash = jQuery(location).attr('hash');
if(hash != '') {
	// check if accordion widget is present
	if(jQuery('.elementor-widget-n-accordion').length > 0) {
     		// open accordion widget
		jQuery('#'+hash).click();
		}
	}
```
